src/main/java/org/cascadebot/cascadebot/commands/music/KaraokeCommand.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2020 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.commands.music;
+
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.Message;
+import org.cascadebot.cascadebot.CascadeBot;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandMain;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.messaging.Messaging;
+import org.cascadebot.cascadebot.music.KaraokeHandler;
+import org.cascadebot.cascadebot.music.MusicHandler;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+
+import javax.xml.parsers.ParserConfigurationException;
+import java.util.concurrent.ExecutionException;
+
+public class KaraokeCommand implements ICommandMain {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        if (context.getMusicPlayer().getPlayer().getPlayingTrack() == null) {
+            KaraokeHandler.disableKaraoke(context.getGuild().getIdLong());
+            context.getTypedMessaging().replyDanger(context.i18n("commands.karaoke.nothing_playing"));
+            return;
+        }
+
+        if (!context.getMusicPlayer()
+                   .getPlayer()
+                   .getPlayingTrack()
+                   .getSourceManager()
+                   .getSourceName()
+                   .equals(CascadeBot.INS.getMusicHandler().getYoutubeSourceName())) {
+            context.getTypedMessaging().replyDanger(context.i18n("commands.karaoke.not_youtube"));
+        }
+
+        if (KaraokeHandler.isKaraoke(context.getGuild().getIdLong())) {
+            KaraokeHandler.disableKaraoke(context.getGuild().getIdLong());
+            context.getTypedMessaging().replySuccess(context.i18n("commands.karaoke.disabled_karaoke"));
+            return;
+        } else {
+            context.getTypedMessaging().replySuccess(context.i18n("commands.karaoke.enabled_karaoke"));
+        }
+        try {
+            Message message = Messaging.sendInfoMessage(context.getChannel(), context.i18n("commands.karaoke.loading_karaoke")).get();
+            KaraokeHandler.getSongLyrics(context.getMusicPlayer().getPlayer().getPlayingTrack().getIdentifier(), context.getChannel(), context.getGuild().getIdLong(), message);
+        } catch (InterruptedException | ExecutionException e) {
+            context.getTypedMessaging().replyDanger("commands.karaoke.cannot_find");
+            CascadeBot.LOGGER.error("Error in karaoke command", e);
```

Comment:
These exceptions would not be reached if we couldn't find the song, it would be for other issues. Therefore, we need to send the exception to the user using the `sendException` method.

Reply:
I'm pretty sure you told me to swallow the error in the first place, but alright.

Where's the sendException method?